### ISSUE TITLE
Installation failed: MAX_TORCH_SEED_VALUE  too high and specific versions for pytorch and fbprophet needed

### ISSUE THREAD

**Issue by JBOE22175** [unit8co/darts#235]
Install darts on windows

1) Following the install guide leads to "ERROR: No matching distribution found for torch==1.5.1; extra == "all" (from u8darts[all])"
You need to specify the versions:
conda install -c conda-forge -c pytorch pip fbprophet==0.6 pytorch==1.5.1  cpuonly 

2) when fitting the model in "TCN-examples.ipynb" following error pops up:
```
File "C:\Users\JB\miniconda3\envs\darts2\lib\site-packages\darts\utils\torch.py", line 62, in decorator
    manual_seed(self._random_instance.randint(0, high=MAX_TORCH_SEED_VALUE))
  File "mtrand.pyx", line 743, in numpy.random.mtrand.RandomState.randint
  File "_bounded_integers.pyx", line 1343, in numpy.random._bounded_integers._rand_int32
ValueError: high is out of bounds for int32
```
I change coding in "...\site-packages\darts\utils\torch.py" in line 17:
```
#MAX_TORCH_SEED_VALUE = (1 << 63) - 1
MAX_TORCH_SEED_VALUE = 2147483647
```
After this change it works. It seems that results of model training is not affected. 

**System (please complete the following information):**
- Win 10
 - Python version:  3.7
 - darts version  0.4.0

**Comment by hrzn** [comment]
Thanks for reporting @JBOE22175

**Comment by vl2376** [comment]
Still having this issue with darts 0.6.1, python 3.6 and Windows 10

**Comment by JBOE22175** [comment]
Yes, still the same.-- Diese Nachricht wurde von meinem Android Mobiltelefon mit WEB.DE Mail gesendet.Am 18.02.21, 10:42 schrieb vl2376 <notifications@github.com>:

   
  Still having this issue with darts 0.6.1, python 3.6 and Windows 10

**Comment by Vaderv** [comment]
I have this issue as well.

**Comment by cosmerodolfo** [comment]
i have the same issue:

![image](https://user-images.githubusercontent.com/34916389/110132937-740f3c80-7daa-11eb-8cca-34bfe5162255.png)

**Comment by Captain-1337** [comment]
This solution should work:

1. Go to the file \anconda\envs\env_name\lib\site-packages\darts\utils\torch.py in your system
2. Find the line MAX_TORCH_SEED_VALUE = (1 << 63) - 1 which should be around line 17
3. Change it to MAX_TORCH_SEED_VALUE = (1 << 31) - 1 so that it fits into 32 bit int.

(credit: https://stackoverflow.com/questions/66514169/when-tcn-is-used-for-prediction-python-there-is-a-valueerror-high-is-out-of-b)

**Comment by hrzn** [comment]
This was fixed in 0.7.0.
